### PR TITLE
Clean up

### PR DESCRIPTION
**Pull request by stereobooster**
Not sure if you will be interested in those changes, if not - close PR

| Load Time | First Byte | Start Render | Speed Index | First Interactive (beta) | Time   | Requests | Bytes In | Time   | Requests | Bytes In | Cost  |
|-----------|------------|--------------|-------------|--------------------------|--------|----------|----------|--------|----------|----------|-------|
| 2.863s    | 1.396s     | 3.203s       | 3425        | > 3.410s                 | 2.863s | 4        | 66 KB    | 3.428s | 7        | 75 KB    | $---- |

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,38 +1,35 @@
...
-    this.props.dispatch(getData('news', page));
+    this.props.dispatch(getData("news", page));
   }
 
   componentWillReceiveProps(nextProps, nextState, context) {
     const newPage = nextProps.match.params.page;
     const page = this.props.match.params.page;
     if (newPage !== page) {
-      this.props.dispatch(getData('news', newPage));
+      this.props.dispatch(getData("news", newPage));
```
Comment (stephenkingsley) [inline src/App.js:16]: I think you should run eslint, please.

**Review by stephenkingsley — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 1 above)
Comment (stereobooster) [reply src/App.js:16]: There is eslint config in CRA, I fixed all warnings generated by CRA. There is no `.eslintrc` in the project root and no npm task in `package.json`. What do you mean?

Comment (stephenkingsley) [reply src/App.js:16]: Please:
1. I want to saw the `.eslintrc` and `precommit` in this project. 
2. Use [airbnb-eslint](https://www.npmjs.com/package/eslint-config-airbnb) is good enough.

Comment (stereobooster) [reply src/App.js:16]: This PR is byproduct of my research about performance improvement. I use CRA's built in eslint configuration and prettier for formatting (which does formatting for me). You have different opinion about formatting and this is ok. What I can do is to use prettier with different configs (tell me what would you like) - I do not have time to configure `eslint` or `precommit`.

If you do not interested in those changes close PR

**Pull request closed without merging**